app/bundles/IntegrationsBundle/EventListener/LeadSubscriber.php

```diff
@@ -199,7 +199,7 @@ private function recordFieldChanges(array $fieldChanges, $objectId, string $obje
                 ->setModifiedAt(new \DateTime())
                 ->setColumnName($key)
                 ->setColumnType($valueDAO->getType())
-                ->setColumnValue($valueDAO->getValue());
+                ->setColumnValue(mb_strlen($valueDAO->getValue()) <= 191 ? $valueDAO->getValue() : substr($valueDAO->getValue(), 0, 191));
```

Comment:
IMO, we should be doing this in the normalization.  And `column_value` is `longtext,` so no need to have this check.

Reply:
@shinde-rahul column_value is varchar 191

![image](https://user-images.githubusercontent.com/462477/125035656-61777d80-e092-11eb-94d5-e1e7ece554ac.png)

Reply:
Probably on your custom branch you have longtext, but in community version there is string. 
@escopecz can you confirm it? We've talking about https://github.com/mautic/mautic/blob/208252c88afa0dd27590b5ae13c8e043651e1010/app/bundles/IntegrationsBundle/Entity/FieldChange.php#L102

Then we should migrate also in community the value to longtext? Can Acquia cherry pick it?